tap-snapshots/test/lib/docs.js.test.cjs

```diff
@@ -2385,6 +2395,7 @@ Array [
   "init-module",
   "init-type",
   "init-version",
+  "init-private",
```

Comment:
it seems kind of weird to have `init-foo` when we also have `init.foo`. why not `init.private`?

Reply:
The dot notated versions of the configs are legacy/deprecated/going away. https://github.com/npm/cli/blob/4b08e2ed252a18f85a360b76c7273a7aa7a994ca/workspaces/config/lib/definitions/definitions.js#L974

Reply:
The dot notated versions are deprecated and we will not be adding new ones

https://github.com/npm/cli/blob/4b08e2ed252a18f85a360b76c7273a7aa7a994ca/workspaces/config/lib/definitions/definitions.js#L974

Reply:
ah k, that makes sense.